### DISCUSSION TITLE
Are there DDEV stickers?

### DISCUSSION THREAD

**Discussion by ChloeSartorelli** [ddev/ddev#5717] [General]
Bit of a random question, but I was wondering if there's an official way to get DDEV stickers?
I'd love to get maybe a sticker of the logo to stick on my laptop.

**Comment by rfay** [comment]
We have had stickers! Gave out some that somebody discovered at Drupalcon Pittsburgh, but that was the end of them. We could make more of course. I have one of the last ones on my laptop.

You absolutely deserve a DDEV sticker!

**Reply by rfay** [reply]
Want to get some made? Should be easy. Logo info at https://ddev.readthedocs.io/en/latest/developers/brand-guide/#logos

**Reply by ChloeSartorelli** [reply]
Ahh okay. Yeah I can get a few made up. I just wanted to get them through an official way if there was already something set up - I know some projects have arrangements with specific sticker vendors to make them on demand so I wondered if DDEV had something similar.

**Reply by rfay** [reply]
It would be lovely if you got some made and got them to some conferences and such. @bmartinez287 has been working on t-shirts as well, over in Discord.

**Reply by bmartinez287** [reply]
I had not looked into stickers since people usually buy boxes of those. That said, It should be pretty simple to add to the threadless market store, I will start prototyping on it. The shirts have worked well from them so I imagine their stickers should be fine.

**Reply by rfay** [reply]
@GuySartorelli if you want stickers I'll be happy to snail-mail some to you! @bmartinez287 gave me a pile of them at Drupalcamp Florida. Just say how many you think you want.

**Reply by ChloeSartorelli** [reply]
Oooh yes please, that would be incredible. I only need 1, but if you have up to 40 I could share them with people at work and build some hype.

**Reply by rfay** [reply]
Very cool, PM me other otherwise send snail-mail address and I'll do it. If email is easiest, do `support (at) ddev.com`

**Reply by ChloeSartorelli** [reply]
Email sent, thanks again!

**Reply by ChloeSartorelli** [reply]
They arrived today. Thanks again!

**Reply by rfay** [reply]
Yay!
